src/elastio-snap.c

```diff
@@ -3723,6 +3767,8 @@ static int snap_cow_thread(void *data){
 		}
 	}
 
+	LOG_DEBUG("snap_cow_thread() done.");
```

Comment:
This is not an abandoned print, I'd like to keep it for debugging in the future.